conf/docker/dataverse-glassfish/Dockerfile

```diff
@@ -57,7 +59,63 @@ RUN chgrp -R 0 /usr/local/glassfish4 && \
 
 #RUN exitEarlyBeforeJq
 RUN yum -y install epel-release
-RUN yum install -y jq 
+# RUN rpm --rebuilddb 
+RUN yum -y install jq 
+
+#JHOVE
+RUN cp /tmp/jhove* /usr/local/glassfish4/glassfish/domains/domain1/config
+
+
+#SETUP JVM OPTIONS 
+RUN cd /tmp \
+    && cd /usr/local/glassfish4/glassfish/bin \
+    && ./asadmin start-domain \
+    &&./asadmin $ASADMIN_OPTS delete-jvm-options "-XX\:MaxPermSize=192m" \
+    && ./asadmin $ASADMIN_OPTS create-jvm-options "-XX\:MaxPermSize=512m"\
+    && ./asadmin $ASADMIN_OPTS create-jvm-options "-XX\:PermSize=256m"\
+    && ./asadmin $ASADMIN_OPTS delete-jvm-options -client \
+    && ./asadmin $ASADMIN_OPTS create-jvm-options "\-Ddataverse.files.directory=/usr/local/glassfish4/glassfish/domains/domain1/files"\
+    # Rserve-related JVM options: 
+    && ./asadmin $ASADMIN_OPTS create-jvm-options "\-Ddataverse.rserve.host=localhost" \
+     && ./asadmin $ASADMIN_OPTS create-jvm-options "\-Ddataverse.rserve.port=6311" \
+    && ./asadmin $ASADMIN_OPTS create-jvm-options "\-Ddataverse.rserve.user=rserve" \
+    && ./asadmin $ASADMIN_OPTS create-jvm-options "\-Ddataverse.rserve.password=rserve" \
+    # Data Deposit API options
+    && ./asadmin $ASADMIN_OPTS create-jvm-options "\-Ddataverse.fqdn='localhost:8080'" \
```

Comment:
dataverse.fqdn (the external hostname of the dataverse installation) is probably something that should be configurable; but that may be out of scope for this issue/PR